app/controllers/teams/users_controller.rb

```diff
@@ -1,5 +1,33 @@
 class Teams::UsersController < AuthenticatedController
-   before_action -> { authorize team, policy_class: TeamUsersPolicy }
+    before_action -> { authorize team, policy_class: TeamUsersPolicy }
+
+    def new
+      @user = User.new
+    end
+  
+    def create
+      add_email = user_params.require(:email).downcase
+      if not valid_email_address?(add_email)
+        @user = User.new
```

Comment:
`@user` doesn't appear to be used in any of the fail states, what's the reason for it being set?